docs/tutorial/index.md

```diff
@@ -1,74 +1,48 @@
-# Get Started with UP4W
+# Get started with UP4W
 
-In this tutorial you will learn all the basic things that you need to know to start taking advantage of your Ubuntu Pro subscription on Ubuntu WSL at scale with Ubuntu Pro for WSL (UP4W).
+Windows Subsystem for Linux ([WSL](https://ubuntu.com/desktop/wsl)) makes it possible to run Ubuntu — the number 1 open-source operating system — on a Windows machine.
+With Ubuntu Pro for WSL (UP4W) an [Ubuntu Pro](https://ubuntu.com/pro) subscription empowers you to manage Ubuntu WSL instances at scale.
 
-<br/>
-
-**What you'll need:**
-
-- A Windows 11 machine with access to the Microsoft Store and a minimum of 16GB of RAM and 8-core processor.
-
-- Familiarity with the Windows PowerShell.
-
-<br/>
-
-**What you'll do:**
-
-- Set things up
-
-- Watch UP4W transform your Ubuntu Pro game on WSL!
-
-- Tear things down
-
-<br/>
+In this tutorial you will develop an understanding of how UP4W can be installed and deployed for managing multiple WSL instances.
+To complete this tutorial you will need a Windows 11 machine with a minimum of 16GB RAM and a 8-core processor.
 
 ```{note}
-Throughout this tutorial we'll present shell commands together with their output:
-
-- The commands will be prefixed by a
-prompt matching the shell it's launched in. 
-
-  For example, `PS C:\Users\me\tutorial>` is a PowerShell prompt where the current working directory is `C:\Users\me\tutorial`, thus running on Windows.
+WSL enables using a Linux shell and Windows PowerShell side-by-side on the same machine.
+Throughout this tutorial, commands will be prefixed by a prompt that indicates the shell being used, for example:
 
+- `PS C:\Users\me\tutorial>` is a PowerShell prompt where the current working directory is `C:\Users\me\tutorial`.
 
-  Similarly, a Linux shell prompt logged in as root with the current working directory as the root home directory will look like `root@hostname:~#`
+- `root@hostname:~#` indicates a Linux shell prompt with login as root in the root home directory `~`.
```

Comment:
I will think about this and any implications for the rest of tutorial. We need to be careful about prompts and prompt symbols in general.
Typically it's not necessary to include prompts or prompt symbols and it can actually cause issues for some users who inadvertently copy/paste them.
In this tutorial it is probably necessary to distinguish between different shells and instances.

Using `/home/ubuntu/` instead of `~` could be a good option also.

Reply:
I agree, maybe just > and $ to differentiate Windows and Linux. Or just no prompt but then it's difficult to make the difference between the OS and what is a command and an output of a command.